src/redux/actions/profile/__test__/update-profile.spec.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable prefer-promise-reject-errors */
```

Comment:
this action have problems, it is not working correctly so we need to reimplement